specs/protocol/holocene/exec-engine.md

```diff
@@ -73,5 +81,73 @@ an outbound withdrawal for a long period of time, the node may not have access t
 clients are able to at the very least reconstruct the account storage root at a given block on the fly if it does not
 directly store this information.
 
+## Extended `PayloadAttributesV3`
+
+The [`PayloadAttributesV3`](https://github.com/ethereum/execution-apis/blob/cea7eeb642052f4c2e03449dc48296def4aafc24/src/engine/cancun.md#payloadattributesv3)
+type is extended to:
+
+```rs
+PayloadAttributesV3: {
+    timestamp: QUANTITY
+    random: DATA (32 bytes)
+    suggestedFeeRecipient: DATA (20 bytes)
+    withdrawals: array of WithdrawalV1
+    parentBeaconBlockRoot: DATA (32 bytes)
+    transactions: array of DATA
+    noTxPool: bool
+    gasLimit: QUANTITY or null
+    eip1559Params: QUANTITY or null
+}
+```
+
+### `eip1559Params` encoding
+
+| Name          | Type  | Byte Offset |
+| ------------- | ----- | ----------- |
+| `denominator` | `u32` | `[0, 4)`    |
+| `elasticity`  | `u32` | `[4, 8)`    |
```

Comment:
Previously these were encoded as `uint64` types. Why are we reducing to `u32`?

Reply:
I see this is to fit in the u64 nonce field. Maybe we can add this as context?

Reply:
maybe since the field is limited to 32 bytes we should be frugal in case we want to put more things here in the future?  We don't need a lot of precision for either of these.

Reply:
You probably want to update the configurability.md to also specify 32 byte encoding in the configupdate event type?  Also while you're there can you change "enum" to "event"?

https://github.com/ethereum-optimism/optimism/pull/12220